### PR TITLE
Fire check before task delay (v3)

### LINKED ISSUE #10: Periodic checks don't fire immediately but instead after the specified interval
As of v6 the behavior changed from calling periodic checks immediately and then with the specified interval.

https://github.com/Particular/NServiceBus.CustomChecks/commit/a1fa779b77542b9805fd87fdd746c83cee5b9d16

This feels off (got bitten by this my self testing the SC dlq checks) and seems to contradict or docs that states 

>All custom checks are executed when the endpoint starts up. If the optional interval is specified then the custom check will be executed periodically.

https://docs.particular.net/monitoring/custom-checks/writing-custom-checks

### PR DESCRIPTION
**Pull request by boblangley**
Fixes #10

### CODE REVIEW

File: src/NServiceBus.CustomChecks.Tests/NServiceBus.CustomChecks.Tests.csproj [hunk 1]
```diff
@@ -12,14 +12,12 @@
...
 
   <ItemGroup>
-    <PackageReference Include="NServiceBus" Version="7.0.0" />
-  </ItemGroup>
-
-  <ItemGroup>
+    <PackageReference Include="NServiceBus" Version="7.1.1" />
     <PackageReference Include="ApprovalTests" Version="3.0.13" NoWarn="NU1701" />
     <PackageReference Include="ApprovalUtilities" Version="3.0.13" NoWarn="NU1701" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0" />
```
Comment (bording) [inline src/NServiceBus.CustomChecks.Tests/NServiceBus.CustomChecks.Tests.csproj]: Microsoft.NET.Test.Sdk should be updated to the latest version.

File: src/NServiceBus.CustomChecks.Tests/NServiceBus.CustomChecks.Tests.csproj [hunk 2]
```diff
@@ -12,14 +12,14 @@
...
   <ItemGroup>
-    <PackageReference Include="NServiceBus" Version="7.1.1" />
+    <PackageReference Include="NServiceBus" Version="7.1.5" />
     <PackageReference Include="ApprovalTests" Version="3.0.13" NoWarn="NU1701" />
     <PackageReference Include="ApprovalUtilities" Version="3.0.13" NoWarn="NU1701" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0" />
-    <PackageReference Include="NUnit" Version="3.10.1" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
+    <PackageReference Include="NUnit" Version="3.11.0" />
     <!-- requires a version that works with PublicApiGenerator (Mono.Cecil) -->
```
Comment (bording) [inline src/NServiceBus.CustomChecks.Tests/NServiceBus.CustomChecks.Tests.csproj]: This comment should be removed now!

File: src/NServiceBus.CustomChecks/NServiceBus.CustomChecks.csproj [hunk 3]
```diff
@@ -15,12 +15,11 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Particular.CodeRules" Version="0.2.0" PrivateAssets="All" />
-    <PackageReference Include="Particular.Packaging" Version="0.1.0" PrivateAssets="All" />
+    <PackageReference Include="Particular.CodeRules" Version="0.2.1" PrivateAssets="All" />
+    <PackageReference Include="Particular.Packaging" Version="0.2.1" PrivateAssets="All" />
   </ItemGroup>
 
   <ItemGroup>
```
Comment (bording) [inline src/NServiceBus.CustomChecks/NServiceBus.CustomChecks.csproj]: This entire ItemGroup should be removed.

File: src/NServiceBus.CustomChecks.Tests/NServiceBus.CustomChecks.Tests.csproj (hunk 2 above)
Comment (boblangley) [reply src/NServiceBus.CustomChecks.Tests/NServiceBus.CustomChecks.Tests.csproj]: Fine.. geez, so angry 😄

**Comment by bording** [comment]
You could go ahead and switch to Particular.Approvals.

**Pull request merged**